<!doctype html>
<html class="no-js" lang="en">
<head>
	<meta charset="utf-8">
	<meta name="viewport" content="width=device-width, initial-scale=1.0">
	<title>News</title>

    <link rel="stylesheet" type="text/css" href="http://localhost:4000/assets/css/styles_feeling_responsive.css">

  

	<script src="http://localhost:4000/assets/js/modernizr.min.js"></script>

	<script src="https://ajax.googleapis.com/ajax/libs/webfont/1.5.18/webfont.js"></script>
	<script>
		WebFont.load({
			google: {
				families: [ 'Lato:400,700,400italic:latin', 'Volkhov::latin' ]
			}
		});
	</script>

	<noscript>
		<link href='http://fonts.googleapis.com/css?family=Lato:400,700,400italic%7CVolkhov' rel='stylesheet' type='text/css'>
	</noscript>


	<!-- Search Engine Optimization -->
	<meta name="description" content="Lab website of Melissa Lehti-Shiu and Shin-Han Shiu, Professors at Michigan State Universtity">
	
	
	
	
	
	
	<link rel="canonical" href="http://localhost:4000/about_news/">


	<!-- Facebook Open Graph -->
	<meta property="og:title" content="News">
	<meta property="og:description" content="Lab website of Melissa Lehti-Shiu and Shin-Han Shiu, Professors at Michigan State Universtity">
	<meta property="og:url" content="http://localhost:4000/about_news/">
	<meta property="og:locale" content="en_EN">
	<meta property="og:type" content="website">
	<meta property="og:site_name" content="Shiu Lab">
	
	


	

	<link type="text/plain" rel="author" href="http://localhost:4000/humans.txt">

	

	

	<link rel="icon" sizes="32x32" href="http://localhost:4000/assets/img/apple-touch-icon-32.png">

	<link rel="icon" sizes="192x192" href="http://localhost:4000/assets/img/apple-touch-icon-192.png">

	<link rel="apple-touch-icon-precomposed" sizes="180x180" href="http://localhost:4000/assets/img/apple-touch-icon-180.png">

	<link rel="apple-touch-icon-precomposed" sizes="152x152" href="http://localhost:4000/assets/img/apple-touch-icon-152.png">

	<link rel="apple-touch-icon-precomposed" sizes="144x144" href="http://localhost:4000/assets/img/apple-touch-icon-144.png">

	<link rel="apple-touch-icon-precomposed" sizes="120x120" href="http://localhost:4000/assets/img/apple-touch-icon-120.png">

	<link rel="apple-touch-icon-precomposed" sizes="114x114" href="http://localhost:4000/assets/img/apple-touch-icon-114.png">

	
	<link rel="apple-touch-icon-precomposed" sizes="76x76" href="http://localhost:4000/assets/img/apple-touch-icon-76.png">

	<link rel="apple-touch-icon-precomposed" sizes="72x72" href="http://localhost:4000/assets/img/apple-touch-icon-72.png">

	<link rel="apple-touch-icon-precomposed" href="http://localhost:4000/assets/img/apple-touch-icon.png">	

	<meta name="msapplication-TileImage" content="http://localhost:4000/assets/img/apple-touch-icon-144.png">

	<meta name="msapplication-TileColor" content="#fabb00">


	

</head>
<body id="top-of-page" class="post">
	
	
<div id="navigation" class="sticky">
  <nav class="top-bar" role="navigation" data-topbar>
    <ul class="title-area">
      <li class="name">
      <h1 class="show-for-small-only"><a href="http://localhost:4000" class="icon-tree"> Shiu Lab</a></h1>
    </li>
       <!-- Remove the class "menu-icon" to get rid of menu icon. Take out "Menu" to just have icon alone -->
      <li class="toggle-topbar menu-icon"><a href="#"><span>Nav</span></a></li>
    </ul>
    <section class="top-bar-section">

      <ul class="right">
        

              

          
          
        

              

          
          
        

              

          
          
        

              

          
          
        

              

          
          
        

              

          
          
        

              

          
          
            
            
              <li class="divider"></li>
              <li><a  href="http://localhost:4000/contact/">Contact</a></li>

            
            
          
        
        
      </ul>

      <ul class="left">
        

              

          
          

            
            
              <li><a  href="http://localhost:4000/">Home</a></li>
              <li class="divider"></li>

            
            
          
        

              

          
          

            
            

              <li class="has-dropdown">
                <a  href="http://localhost:4000/about_mission/">About</a>

                  <ul class="dropdown">
                    

                      

                      <li><a  href="http://localhost:4000/about_mission/">Mission</a></li>
                    

                      

                      <li><a  href="http://localhost:4000/about_dei/">Diversity, equity, &amp; inclusion</a></li>
                    

                      

                      <li><a  href="http://localhost:4000/about_news/">News</a></li>
                    
                  </ul>

              </li>
              <li class="divider"></li>
            
          
        

              

          
          

            
            

              <li class="has-dropdown">
                <a  href="http://localhost:4000/people_current/">People</a>

                  <ul class="dropdown">
                    

                      

                      <li><a  href="http://localhost:4000/people_current/">Current</a></li>
                    

                      

                      <li><a  href="http://localhost:4000/people_past/">Past</a></li>
                    

                      

                      <li><a  href="http://localhost:4000/people_join_us/">Join Us!</a></li>
                    
                  </ul>

              </li>
              <li class="divider"></li>
            
          
        

              

          
          

            
            

              <li class="has-dropdown">
                <a  href="http://localhost:4000/research_projects/">Research</a>

                  <ul class="dropdown">
                    

                      

                      <li><a  href="http://localhost:4000/research_projects/">Projects</a></li>
                    

                      

                      <li><a  href="https://github.com/ShiuLab">Software</a></li>
                    
                  </ul>

              </li>
              <li class="divider"></li>
            
          
        

              

          
          

            
            

              <li class="has-dropdown">
                <a  href="http://localhost:4000/publications_last5/">Publications</a>

                  <ul class="dropdown">
                    

                      

                      <li><a  href="http://localhost:4000/publications_last5/">last 5 years</a></li>
                    

                      

                      <li><a  href="http://localhost:4000/publications_older/">More than 5 years</a></li>
                    
                  </ul>

              </li>
              <li class="divider"></li>
            
          
        

              

          
          

            
            

              <li class="has-dropdown">
                <a  href="http://localhost:4000/activities_catsup/">Activities</a>

                  <ul class="dropdown">
                    

                      

                      <li><a  href="http://localhost:4000/activities_catsup/">CATSUP</a></li>
                    

                      

                      <li><a  href="http://localhost:4000/activities_social/">Social</a></li>
                    

                      

                      <li><a  href="http://localhost:4000/activities_outreach/">Outreach</a></li>
                    
                  </ul>

              </li>
              <li class="divider"></li>
            
          
        

              

          
          
        
        
      </ul>
    </section>
  </nav>
</div><!-- /#navigation -->

	

	

<div id="masthead">
	<div class="row">
		<div class="small-12 columns">
			<a id="logo" href="http://localhost:4000/" title="Shiu Lab – Computational, Evolutionary, and Predictive Biology Lab">
				<img src="http://localhost:4000/assets/img/logo.png" alt="Shiu Lab – Computational, Evolutionary, and Predictive Biology Lab">
			</a>
		</div><!-- /.small-12.columns -->
	</div><!-- /.row -->
</div><!-- /#masthead -->










	<div class="row t30">
	<div class="medium-8 columns medium-offset-2 end">
		<article itemscope itemtype="http://schema.org/Article">
			<header>
				

				<div itemprop="name">
					<p class="subheadline">About</p>
					<h1>News</h1>
				</div>
			</header>


			

			<div itemprop="articleSection">
			<h4 id="72921-siobhans-study-on-modeling-plant-genetic-redundancy-is-published">7/29/21: Siobhan’s study on modeling plant genetic redundancy is published</h4>

<p><img src="/images/news/210729_siobhan_genetic_redundancy.jpg" width="100" align="right" />Genetic redundancy refers to a situation where an individual with a loss-of-function mutation in one gene (single mutant) does not show an apparent phenotype until one or more paralogs are also knocked out (double/higher-order mutant). Siobhan led a study aiming at understanding the factors contributing to genetic redundancy with machine learning approaches. In addition to Siobhan, Peipei, Serena, Beth, Fanrui, and Melissa have contributed significantly to the study. This work is now <a href="https://pubmed.ncbi.nlm.nih.gov/33871641/">published in Molecular Biology &amp; Evolution.</a></p>

<h4 id="72021-an-essay-on-overcoming-challenges-to-enhancing-experimental-biology-with-computational-modeling-is-published">7/20/21: An essay on overcoming challenges to enhancing experimental biology with computational modeling is published</h4>

<p><img src="/images/news/210720_experimental_biology_essay.jpg" width="100" align="right" />Computational modeling approaches are underutilized in plant biology. Lead by Renee Dale from Danforth Plant Science Center, Shinhan and seven other colleagues <a href="https://pubmed.ncbi.nlm.nih.gov/34354723/">published an opinion piece in Frontiers in Plant Science</a> identifying challenges one may face in adopting modeling approach.</p>

<h4 id="41721-why-are-there-no-symptoms-when-a-plant-is-infected-with-a-fungal-pathogen">4/17/21: Why are there no symptoms when a plant is infected with a fungal pathogen?</h4>

<p><img src="/images/news/210417_shinhan_fungal_pathogen.gif" width="100" align="right" />The broad host range of Fusarium virguliforme represents a unique comparative system to identify and define differentially induced responses between an asymptomatic monocot host, maize (Zea mays), and a symptomatic eudicot host, soybean (Glycine max). Led by Amy Baetsen-Young from Brad Day’s lab, Shinhan collaborated on a project examining how asymptomatic and symptomatic hosts respond to F. virguliforme transcriptionally. This work is now <a href="https://pubmed.ncbi.nlm.nih.gov/33681966/">published in Plant Cell</a></p>

<h4 id="12720-welcome-the-newest-member-of-our-lab---kenia-sugura-abá">12/7/20: Welcome the newest member of our lab - Kenia Sugura Abá</h4>

<p><img src="/images/news/201210_kenia_joining_lab.jpg" width="100" hspace="30" align="right" />Kenia is a first year graduate student in the Genetics and Genomic Science Program. She rotated over the summer and has made the decision to join our lab today! She graduated from University of Texa-Austin with families in the San Antonio area. Welcome to our lab Kenia!</p>

<h4 id="9120-collaborative-work-on-maize-ant1-regulatory-network-published">9/1/20: Collaborative work on maize ANT1 regulatory network published!</h4>

<p>Arabidopsis AINTEGUMENTA (ANT), an AP2 transcription factor, is known to control plant growth and floral organogenesis. This study is led by Dr. Wen-Hsiung Li from Academia Sinica, Taiwan, which reveals biological roles of ANT1 in several developmental processes beyond its known roles in plant growth and floral organogenesis. It is published in <a href="https://pubmed-ncbi-nlm-nih-gov.proxy1.cl.msu.edu/32817425/">the Sep. 1st issue of Proc. Natl. Acad. Sci., USA</a>.</p>

<h4 id="9120-christinas-story-on-predictions-of-plant-stress-response-through-multi-omics-data-integration-is-published">9/1/20: Christina’s story on predictions of plant stress response through multi-omics data integration is published!</h4>

<p>Plants respond to their environment by dynamically modulating gene expression. Our findings demonstrate how in silico approaches can improve our understanding of the complex codes regulating response to combined stress and help us identify prime targets for future characterization. It is published in <a href="https://academic.oup.com/nargab/article/2/3/lqaa049/5874366">Nuc. Acid. Res. Genomics and Bioinformatics</a>.</p>

<h4 id="73020-beths-story-on-using-arabidopsis-annotation-information-to-improve-predictions-of-tomato-metabolic-genes-is-published">7/30/20: Beth’s story on using Arabidopsis annotation information to improve predictions of tomato metabolic genes is published!</h4>

<p>Plant specialized metabolites mediate interactions between plants and the environment and have significant agronomical/pharmaceutical value. This study demonstrates that specialized and general metabolic genes can be better predicted by leveraging cross-species information. Additionally, our findings provide an example for transfer learning in genomics where knowledge can be transferred from an information-rich species to an information-poor one. The findings is now available form <a href="https://doi.org/10.1093/insilicoplants/diaa005">In Silico Plant</a>.</p>

<h4 id="62520-serena-won">6/25/20: Serena won!!</h4>

<p><img src="/images/news/200620_serena_small.png" width="100" hspace="30" align="right" /> Serena is the winner of the SciComm Blog Contest! Here is her winning post <a href="https://www.msuscicomm.org/post/knowledge-graphs">KNOWLEDGE GRAPHS: A New Way To Reason</a>. Congrats Serena!</p>

<h4 id="62120-virtual-lunch-in-its-3rd-month-now-with-fanrui">6/21/20: Virtual lunch in its 3rd month… Now with Fanrui!!</h4>

<p><img src="/images/news/200621_fanrui_small.png" width="100" hspace="30" align="right" />Fanrui, our deep learning expert, Peipei’s beloved husband, and a great cook, is finally back after getting stuck in China for &gt;6 months due to the COVID19, repeated flight canceldation, and long stay in the secondary screen through the immigration. Welcome back!</p>

<h4 id="61120-kevin-allys-brother-says-hi">6/11/20: Kevin, Ally’s brother, says hi!</h4>

<p><img src="/images/news/200611_kevin_says_hi.png" width="100" hspace="30" align="right" />Kevin likes to draw/color while Ally does work. During our lab reading group meeting, Kevin decided to draw a picture for all of us. Here it is!
<br />
<br /></p>

<h4 id="51320-virtual-lab-lunch-at-its-second-month">5/13/20: Virtual lab lunch at its second month</h4>

<p><img src="/images/news/200513_lab-lunch.png" width="300" hspace="30" align="right" />In our lab lunch today, Ronan is biting her mom’s face!
<br />
<br />
<br /></p>

<h4 id="51120-christinas-review-on-interpretable-machine-learning-is-published">5/11/20: Christina’s review on interpretable machine learning is published</h4>

<p><img src="/images/200601_chrisitna_tig_cover.jpg" width="100" hspace="30" align="right" />In collaboration with Jiliang Tang from Comp. Sci. &amp; Engr, MSU, Christina’s review on <a href="https://www.ncbi.nlm.nih.gov/pubmed/32396837"><strong>Opening the Black Box: Interpretable Machine Learning for Geneticists</strong></a> is now published in Trends in Genetics. This work highlights current progress in making machine learning model interpretable and explanable using applications in genetics as examples. And her graphics was picked as the cover as shown on the right!</p>

<h4 id="4820-virtual-lab-lunch-for-almost-a-month">4/8/20: Virtual lab lunch for almost a month</h4>

<p><img src="/images/news/200408_lab-lunch.png" width="300" hspace="30" align="right" />With the social distancing in place, we continue to meet over Zoom. It seems that folks are doing ok!</p>

<h4 id="33120-collaborative-work-with-dr-yan-bao-is-published">3/31/20: Collaborative work with Dr. Yan Bao is published</h4>

<p>Dr. Bao’s work on <a href="https://www.ncbi.nlm.nih.gov/pubmed/32170020"><strong>COST1 regulates autophagy to control plant drought tolerance</strong></a> is published where Peipei has contributed significantly using comparative genomics approaches.</p>

<h4 id="31920-siobhan-defended-today">3/19/20: Siobhan defended today</h4>

<p><img src="/images/news/200319_siobahn-before-defense.png" width="100" hspace="30" align="right" />Siobahn defended her thesis titled <em>Modeling and prediction of genetic redundancy in Arabidopsis thaliana and Saccharomyces cerevisiae</em> today <strong>virtually</strong> via Zoom with 89 people in the audience! Both us showed up in the lab (while keeping &gt;6 ft distance) to get the virtual defense setup. Originally due to move to England next week, Siobhan will stay in East Lansing to finish her work. Good job Siobhan! The picture on the right shows Siobhan right after the defense.</p>

<h4 id="31320-our-first-virtual-lab-lunch">3/13/20: Our first virtual lab lunch</h4>

<p><img src="/images/news/200325_lab-lunch.png" width="200" hspace="30" align="right" />With all of us working remotely, we decide to host a 15-min lab lunch to check in with everyone. The photo on the right is all of us experimenting with the background feature in Zoom - the two “ghosts” are Siobhan and Serena!</p>

<h4 id="31120-social-distancing-starts">3/11/20: Social distancing starts</h4>

<p>Michigan State has indicated all classes will be online only today. Given the nature of most of our work is computational and considering the risks of COVID-19, from today on all lab personnel will be working remotely, except for those who need to keep our plant alive.</p>

<h4 id="3520-welcome-to-the-lab-ally">3/5/20: Welcome to the lab Ally</h4>

<p><img src="/images/people/Ally.png" width="100" hspace="30" align="right" />Ally has decided to join the Plant Biology Program and, instead of doing any rotation, is coming to our lab! She is currently finishing up her Bachelor’s degree in Manchester University, a liberal art institution. She is awarded an MSU College of Natural Science Fellowship as well as an Early-Start Fellowship to begin her doctoral training on July 1st! Welcome to our lab Ally!</p>

<h4 id="22120-welcome-to-the-lab-serena">2/21/20: Welcome to the lab Serena</h4>

<p><img src="/images/people/Serena.png" width="100" hspace="30" align="right" />Serena is a new graduate student in the, hold your breath, Plant Biology Program, the Computational Math., Sci., Math Program, AND the Moleculr Plant Science Program. She is a MSU Plant Science Fellow and an NSF Fellow of the IMPACTS training program. She graduated from the Cornell University in 2019 and has been rotating in two other labs. <a href="/images/news/200221_serena-slack.png">Here is the funny Slack exchange</a> when I found out she is joining us!</p>

<h4 id="1120-christinas-study-on-using-genomic-and-transcriptomic-information-to-predict-trait-is-published">1/1/20: Christina’s study on using genomic and transcriptomic information to predict trait is published</h4>

<p>Can transcriptome data be used to predict traits? In the study titled <a href="https://www.ncbi.nlm.nih.gov/pubmed/31641024"><strong>Transcriptome-Based Prediction of Complex Traits in Maize</strong></a>, Christina, in collaboration with Jeremy, a rotating student at the time, and Bob VanBuren tested this and found out that not only we can predict traits with transcriptomes, but they may tell us more than genomic data can. The finding is featured in <a href="https://msutoday.msu.edu/news/2020/scientists-improve-yield-predictions-based-on-seedling-data/">MSU Today</a>!</p>

<h4 id="102519-beth-defended-her-phd">10/25/19: Beth defended her PhD</h4>

<p><img src="/images/news/191025_beth-before-defense.png" width="100" hspace="30" align="right" />Beth defended her PhD thesis today titled <em>Specilized metabolism and stress response: studies in predicting gene function and regulation</em>. The seminar was held in Plant Biology Lab 247 and was attended by both her colleagues, friends, and families. Beth is moving to University of Wisconsin-Madison to take a postdoctoral position applying molecular evolutionary and computaional approaches to plant biochemistry. We wish her all the best! The picture on the right shows Beth right before her defense.</p>

<h4 id="102019-ronan-is-here">10/20/19: Ronan is here!</h4>

<p>Siobhan gave birth today to her baby boy Ronan! Congrats Siobhan and Geoff!</p>

<h4 id="92719-christina-defeneded-her-phd">9/27/19: Christina defeneded her PhD</h4>

<p><img src="/images/news/191003_christina-last-day.png" width="200" hspace="30" align="right" />Christina today defended her PhD thesis titled <em>Interpretable machine laerning in plant genomes: studies in the complex relationship between genotype and phenotype</em>. The seminar was held in Plant Biology Lab 247 and was full! Christina’s parents also attended her seminar. In the talk, Christina laid out the overarching Christina is now moving to Australia working in algorithm development in single cell genomics studies. All the best Christina! The picture on the right showing Chrisitna leaving the lab for good on 10/3/2020…</p>

<h4 id="92419-sahra-and-christinas-work-on-the-regulatory-mechanisms-of-cell-type-stress-response-is-published">9/24/19: Sahra and Christina’s work on the regulatory mechanisms of cell-type stress response is published</h4>

<p>Our story titled <a href="https://www.ncbi.nlm.nih.gov/pubmed/31551359"><strong>Cis-regulatory code for predicting plant cell-type transcriptional response to high salinity</strong></a> is now published in Plant Physiology with Sahra and Christina as joint first authors. Using the root cell-type transcriptome data, Sahra and Christina identified cis-regulatory sequences likely specify cell-type response to high salinity stress. More importantly, machine learning models were built to ask how well the identified regulatory sequences can predict cell-type response. The findings not only advance our understanding of the regulatory mechanisms of the plant spatial transcriptional response, but also suggest broad applicability of the approach to any species, particularly those with little or no trans regulatory data.</p>

<h4 id="91819-christinas-work-on-benchmarking-genomic-prediction-algorithms-is-published">9/18/19: Christina’s work on benchmarking genomic prediction algorithms is published</h4>

<p>Chrisitina’s paper on <a href="https://www.ncbi.nlm.nih.gov/pubmed/31533955"><strong>Benchmarking Parametric and Machine Learning Models for Genomic Prediction of Complex Traits</strong></a> is published today in G3. This is a collaborative between our colleague Gustavo de los Compos, Chrisitna’s internship mentors Andrew McCarren and Mark Roantree, and Emily Bolger a highly productive summer research student. Using data of 18 traits across six plant species with different marker densities and training population sizes, Christina spearheaded a comparison of the performance of six linear and six non-linear algorithms.  The findings highlight that simpler, linear algorithm can beat out more sophiscated machine learning approaches, and the importance of algorithm selection for the prediction of trait values.</p>

<h4 id="82019-johns-work-on-poaceae-intergenic-transcription-is-published">8/20/19: John’s work on poaceae intergenic transcription is published</h4>

<p>John’s paper on <a href="https://www.ncbi.nlm.nih.gov/pubmed/31431676"><strong>Evolutionary characteristics of intergenic transcribed regions indicate rare novel genes and widespread noisy transcription in the Poaceae</strong></a> is published in Scientific Report. Christina as well as Rosalie Sowers, a highly talented undergrad from Penn State also contribute to this work. Extensive transcriptional activity occurring in intergenic regions of genomes has raised the question whether intergenic transcription represents the activity of novel genes or noisy expression. To assess this, John et al. evaluated cross-species and post-duplication sequence and expression conservation of intergenic transcribed regions. This study provides a framework to identify novel genes using comparative transcriptomic data to improve genome annotation that is fundamental for connecting genotype to phenotype in crop and model systems.</p>

<h4 id="81919-our-newest-addition-to-the-lab-thilanka">8/19/19: Our newest addition to the lab, Thilanka</h4>

<p><img src="/images/people/thilanka.jpg" width="100" hspace="30" align="right" />Thilanka is a new graduate student in the Plant Biology Graduate Program. He graduated from the University of Peradeniya in Sri Lanka in 2016 and worked as a teaching assistant in the past two years in Peradeniya. He is energetic, communicative, and excited about learning new topics. He is also easy to get alone with, and has a black belt in Taekwondo!! Welcome to the lab!</p>

<h4 id="72419-siobhan-got-the-dissertation-completion-fellowship">7/24/19: Siobhan got the Dissertation Completion Fellowship</h4>

<p>Siobhan has been selected to receive a $7,500 College of Natural Science Dissertation Completion Fellowship during fall semester 2019. Congrats Siobhan!</p>

<h4 id="71319-nicks-work-on-regulatory-assymmetry-of-plant-duplicate-genes-is-published">7/13/19: Nick’s work on regulatory assymmetry of plant duplicate genes is published</h4>

<p>Nick’s paper on <a href="https://www.ncbi.nlm.nih.gov/pubmed/30866803"><strong>Expression and regulatory asymmetry of retained Arabidopsis thaliana transcription factor genes derived from whole genome duplication</strong></a> is published in BMC Evoltionary Biology. Christina and Eamon Winship, an undergraduate reseracher at the time also contributed. Transcription factors (TFs) play a key role in gene regulation and tend to be retained after duplication. Nick’s work provide answers about about how TF duplicates have diverged in their expression and regulation that contribute to a better understanding of the elevated retention rate among TFs.</p>

<h4 id="21919-our-collaborative-work-on-maize-time-series-transcriptome-is-published">2/19/19: Our collaborative work on maize time-series transcriptome is published</h4>

<p>Led by Dr. Wen-Hsiung Li’s group in Taiwan, Shinhan has contributed to the study: <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=shiu-sh"><strong>Comparative transcriptomics method to infer gene coexpression networks and its applications to maize and rice leaf transcriptomes</strong></a> published in PNAS. This study involves a comprehensive time series data and a novel approach to build time-resolved co-expression network to hypothesize regulatory relationships between genes.</p>

<h4 id="2519-beths-work-on-specialized-metabolism-is-published-and-featured-by-nsf">2/5/19: Beth’s work on specialized metabolism is published and featured by NSF</h4>

<p>Beth’s work on <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=shiu-sh"><strong>Robust predictions of specialized metabolism genes through machine learning</strong></a> is published in PNAS with Peipei, Johnny, Melissa, and multiple colleagues in Rob Last’s group as co-authors. This work features a machine learning approach to predict general and specialized metabolic gene that are notoriously hard to distinquish. The work is featured in <a href="https://msutoday.msu.edu/news/2019/machine-learning-unlocks-plants-secrets/">MSU Today</a> and in <a href="https://www.youtube.com/watch?v=TwI1aeWz9Dc&amp;t=0m38s">NSF’s Awesome Discoveries You Probably Didn’t Hear About This Week</a>.</p>

<h4 id="lots-happened-before-this-point-but-we-transition-to-github-after-early-2019-so">Lots happened before this point.. But we transition to Github after early 2019, so…</h4>

			</div>

			

			
		</article>
	</div><!-- /.medium-8.columns -->


	


	
</div><!-- /.row -->


	
	    <div id="up-to-top" class="row">
      <div class="small-12 columns" style="text-align: right;">
        <a class="iconfont" href="#top-of-page">&#xf108;</a>
      </div><!-- /.small-12.columns -->
    </div><!-- /.row -->


    <footer id="footer-content" class="bg-grau">
      <div id="footer">
        <div class="row">
          <div class="medium-6 large-5 columns">
            <h5 class="shadow-black">About This Site</h5>

            <p class="shadow-black">
              Lab website of Melissa Lehti-Shiu and Shin-Han Shiu, Professors at Michigan State Universtity
              <a href="http://localhost:4000/about_mission/">More ›</a>
            </p>
          </div><!-- /.large-6.columns -->


          <div class="small-6 medium-3 large-3 large-offset-1 columns">
            

              <ul class="no-bullet shadow-black">
              
              </ul>
          </div><!-- /.large-4.columns -->


          <div class="small-6 medium-3 large-3 columns">
            
              
                <h5 class="shadow-black">Useful Links</h5>
              
            
              
            
              
            

            <ul class="no-bullet shadow-black">
            
              
                <li >
                  <a href="http://localhost:4000"  title=""></a>
                </li>
            
              
                <li class="network-entypo" >
                  <a href="https://plantbiology.natsci.msu.edu/" target="_blank"  title="MSU PLB">MSU Plant Biology</a>
                </li>
            
              
                <li class="services-newsletter" >
                  <a href="https://cmse.msu.edu/" target="_blank"  title="MSU CMSE">MSU Comp. Math., Sci. & Engr</a>
                </li>
            
            </ul>
          </div><!-- /.large-3.columns -->
        </div><!-- /.row -->

      </div><!-- /#footer -->


      <div id="subfooter">
        <nav class="row">
          <section id="subfooter-left" class="small-12 medium-6 columns credits">
            <p>Created by <a href="http://phlow.de/">Phlow</a> with <a href="http://jekyllrb.com/" target="_blank">Jekyll</a> based on <a href="http://phlow.github.io/feeling-responsive/">Feeling Responsive</a>.</p>
          </section>

          <section id="subfooter-right" class="small-12 medium-6 columns">
            <ul class="inline-list social-icons">
            
              <li><a href="http://github.com/ShiuLab" target="_blank" class="icon-github" title="Code and more..."></a></li>
            
              <li><a href="https://www.linkedin.com/in/shin-han-shiu-6187b7126/" target="_blank" class="icon-linkedin" title="LinkedIn..."></a></li>
            
              <li><a href="https://scholar.google.com/citations?user=q8qVi6sAAAAJ&hl=en" target="_blank" class="icon-google" title="GoogleScholar..."></a></li>
            
              <li><a href="https://trello.com/shiulab" target="_blank" class="icon-trello" title="LabTrello..."></a></li>
            
            </ul>
          </section>
        </nav>
      </div><!-- /#subfooter -->
    </footer>

	

	


<script src="http://localhost:4000/assets/js/javascript.min.js"></script>



<script>
    $("#masthead").backstretch("https://ShiuLab.github.io/images/header_mps.png", {fade: 700});
    $("#masthead-with-text").backstretch("https://ShiuLab.github.io/images/header_mps.png", {fade: 700});
</script>












</body>
</html>

