---
layout: page
subheadline:  "Publications"
title:  "Recent papers"
header:
   image_fullwidth: "mps_corridor.png"
permalink: "/publications_last5/"
---

[Click here for publications >5 years old](https://shiulab.github.io/publications_older/) or see our [Google Scholar](https://scholar.google.com/citations?user=q8qVi6sAAAAJ&hl=en) page

__Notation__: e: joint first/corresponding, u:undergrad, h: high school student, __Bold__: lab members.

## Pre-prints

__Wang P, Lehti-Shiu MD__, Lotreck S, __Segura Abá K, Shiu SH__. Prediction of plant complex traits via integration of multi-omics data _bioRxiv_ [doi:10.1101/2023.11.14.566971](https://doi.org/10.1101/2023.11.14.566971)

View ORCID ProfilePeipei Wang, Melissa D Lehti-Shiu, Serena Lotreck, Kenia Segura Aba, View ORCID ProfileShin-Han Shiu

__Shiu SH, Lehti-Shiu MD__. Evolution of research topics and paradigms in plant sciences. _bioRxiv_ [doi: 10.1101/2023.10.02.560457](https://www.biorxiv.org/content/10.1101/2023.10.02.560457v1)

## 2023

Lotreck SA, __Segura Abá K, Lehti-Shiu MD, Seeger A<sup>u</sup>, Brown BNI, Ranaweera T__, Schumacher A, Ghassemi M, __Shiu SH__. In a PICKLE: A gold standard entity and relation corpus for the molecular plant sciences. _In Silico Plants_ diad021 [doi](https://doi.org/10.1093/insilicoplants/diad021)

Li Y, Miyani B, Childs KL, __Shiu SH__, and Xagoraraki I. Effect of wastewater collection and concentration methods on assessment of viral diversity. _Sci Total Environ_ 2023:168128 [pubmed](https://pubmed.ncbi.nlm.nih.gov/37918732/)

Lee YS, __Shiu SH__, and Grotewold E. Evolution and diversification of the ACT-like domain associated with plant basic helix–loop–helix transcription factors _Proc. Natl. Acad. Sci. USA_ 120(19):e2219469120 [pubmed](https://pubmed.ncbi.nlm.nih.gov/37126718/)

US DOE. Artificial Intelligence and Machine Learning for Bioenergy Research: Opportunities and Challenges, DOE/SC-0211. U.S. Department of Energy Office of Science and Office of Energy Efficiency and Renewable Energy. [doi](https://doi.org/10.2172/1968870) (Shiu Biodesign: Plant group)

Cuddington K, Abbott KC, Adler FR, Aydeniz M, Dale R, Gross LJ, Hastings A, Hobson EA, Karatayev V, Killion A, Madamanchi A, Marrraffini ML, McCombs A, Samyono W, __Shiu SH__, Watanabe KH, White ER. Challenges and opportunities to build quantitative self-confidence in biologists. _Bioscience_ 73:364 [doi](https://doi.org/10.1093/biosci/biad015)

## 2022

Noble JA, Bielski NV, Liu MJ, DeFalco TA, Stegmann M, Nelson ADL, McNamara K, Sullivan B, Dinh KK, Khuu N, Hancock S, __Shiu SH__, Zipfel C, Cheung AY, Beilstein MA, Palanivelu R. Evolutionary analysis of the LORELEI gene family in plants reveals regulatory subfunctionalization. _Plant Physiol_ 190(4):2539 [pubmed](https://pubmed.ncbi.nlm.nih.gov/36156105/)

**Ranaweera T, Brown BNI, Wang P<font color="green"><sup>e</sup></font>, Shiu SH<font color="green"><sup>e</sup></font>**. Temporal Regulation of Cold Transcriptional Response in Switchgrass. _Frontier in Plant Sci_ 13:998400 [pubmed](https://pubmed.ncbi.nlm.nih.gov/36299783/)

Liu WY, Yu CP, Chang CK, Chen HJ, Li MY, Chen YH, __Shiu SH__, Ku MSB, Tu SL, Lu MJ, Li WH. Regulators of early maize leaf development inferred from transcriptomes of laser capture microdissection (LCM)-isolated embryonic leaf cells. *Proc. Natl. Acad. Sci. USA* 119(35):e2208795119 [pubmed](https://pubmed.ncbi.nlm.nih.gov/36001691/)

Lucker BF, Temple JA, **Panchy NL**, Benning UF, BibikaPeter JD, Neofotis PG, Weissman JC, Baxter IR, __Shiu SH__, Kramer DM. Selection-enriched genomic loci (SEGL) reveals genetic loci for environmental adaptation and photosynthetic productivity in Chlamydomonas reinhardtii. *Algal Res* 64:102709 [doi](https://doi.org/10.1016/j.algal.2022.102709)

van Dijk ADJ, **Shiu SH**, de Ridder D. Editorial: Artificial Intelligence and Machine Learning Applications in Plant Genomics and Genetics. *Front Artif Intell.* 5:959470 [pubmed](https://pubmed.ncbi.nlm.nih.gov/35832206/)

**Wang P<font color="green"><sup>e</sup></font>, Meng F<font color="green"><sup>e</sup></font>, Donaldson P<font color="red"><sup>u</sup></font>, Horan S<font color="red"><sup>u</sup></font>, Panchy  NL, Vischulis E, Winship E<font color="red"><sup>u</sup></font>**, Conner JK, **Shiu SH<font color="green"><sup>e</sup></font>, Lehti-Shiu MD<font color="green"><sup>e</sup></font>**. High throughput measurement of plant fitness traits with an object detection method using Faster R-CNN. *New Phytologist* 234:1521–1533 [pubmed](https://pubmed.ncbi.nlm.nih.gov/35218008/)

**Wang P<font color="green"><sup>e</sup></font>, Schumacher A and Shiu SH<font color="green"><sup>e</sup></font>**. Computational prediction of plant metabolic pathways. *Current Opinion in Plant Biology* 66:102171 [pubmed](https://pubmed.ncbi.nlm.nih.gov/35078130/)

__Moore BM__, Lee Y S, __Wang PP, Azodi C__, Grotewold E, __Shiu SH__. Modeling temporal and hormonal regulation of plant transcriptional response to wounding. *Plant Cell* 34(2):867-888 [pubmed](https://pubmed.ncbi.nlm.nih.gov/34865154/)

Noble JA, __Seddon A, Uygun S__, Bright A, Smith SE, __Shiu SH__, Palanivelu R. The SEEL motif and members of the MYB-related REVEILLE transcription factor family are important for the expression of LORELEI in the synergid cells of the Arabidopsis female gametophyte. *Plant Reproduction* 35(1):61-76 [pubmed](https://pubmed.ncbi.nlm.nih.gov/34716496/)

## 2021

Dale R, Oswald S, Jalihal A, Laporte M, Fletcher DM, Hubbard A, __Shiu SH__, Nelson ADL, Bucksch A. Overcoming the challenges to enhancing experimental plant biology with computational modeling. *Frontier in Plant Science* 12:687652 [pubmed](https://pubmed.ncbi.nlm.nih.gov/34354723/)

**Cusack SA, Wang P, Lotreck SG, Moore BM, Meng F**, Conner JK, Krysan PJ, **Lehti-Shiu MD, Shiu SH**. Predictive models of genetic redundancy in Arabidopsis thaliana. *Molecular Biology and Evolution* 38(8):3397-3414 [pubmed](https://pubmed.ncbi.nlm.nih.gov/33871641/)

**Wang P, Moore BM, Ugyun S, Lehti-Shiu M**, Barry C, **Shiu SH**. 	Optimizing the use of gene expression data to predict plant metabolic pathway memberships. *New Phytologist* 231(1):475-489 [pubmed](https://pubmed.ncbi.nlm.nih.gov/33749860/)

**Wang PP, Meng FR, Moore BM, Shiu SH**. Impact of short-read sequencing on the misassembly of a plant genome. *BMC Genomics* 22(1):99 [pubmed](https://pubmed.ncbi.nlm.nih.gov/33530937/)

Baetsen-Young A, Chen H, **Shiu SH**, Day B. Contrasting transcriptional responses to Fusarium virguliforme infection in symptomatic and asymptomatic hosts. *Plant Cell* 17;33(2):224-247 [pubmed](https://pubmed.ncbi.nlm.nih.gov/33681966/)

## 2020

Liu WY, Lin HH, Yu CP, Chang CK, Chen HJ, Lin JY, Lu MYJ, Tu SL, **Shiu SH** Wu SH, Ku MSB, Li WH. Maize ANT1 modulates vascular development, chloroplast development, photosynthesis and plant growth. *Proc. Natl. Acad. Sci., USA* 117(35):21747 [pubmed](https://pubmed.ncbi.nlm.nih.gov/32817425/)

**Moore BM**, **Wang P**, Fan P, **Lee A**<font color="red"><sup>u</sup></font>, Leong B, Lou YR, Schenck C, Sugimoto K, Last R, **Lehti-Shiu MD**, Barry CS, **Shiu SH**. Within and cross species predictions of plant specialized metabolism genes using transfer learning. *In Silico Plants* 2(1):diaa005 [doi](https://doi.org/10.1093/insilicoplants/diaa005)

**Azodi CB, Lloyd JP, Shiu SH**. The cis-regulatory codes of response to combined heat and drought stress in Arabidopsis thaliana. *Nuc Acid Res-Genomics & Bioinformatics* 2(3):lqaa049 [doi](https://doi-org.proxy2.cl.msu.edu/10.1093/nargab/lqaa049)

Fan P, **Wang P**, Lou YR, Leong BJ, **Moore BM**, Schenck CA, Combs R, Cao P, Brandizzi F, **Shiu SH**, Last RL. Evolution of a plant gene cluster in Solanaceae and emergence of metabolic diversity. *eLife* 9:e56717 [pubmed](https://pubmed.ncbi.nlm.nih.gov/32613943/)

**Azodi CB**, Tang J, **Shiu SH**. Opening the black box: interpretable machine learning for geneticists. <font color="blue">[review]</font> *Trends in Genetics* 36(6):442 [pubmed](https://pubmed.ncbi.nlm.nih.gov/32396837/)

Bao Y, Song W, **Wang P**, Yu X, Li B, Jiang C, **Shiu SH**, Zhang H, Bassham DC (2020) The plant-specific COST1 protein balances plant growth and drought tolerance via attenuation of autophagy. *Proc. Natl. Acad. Sci. USA* 117:7482 [pumbed](https://pubmed.ncbi.nlm.nih.gov/32170020)

**Panchy NL**, **Lloyd JP**, **Shiu SH** (2020) Improved recovery of cell-cycle gene expression in *Saccharomyces cerevisiae* from regulatory interactions in multiple omics data. *BMC Genomics* 21:159 [pubmed](https://pubmed.ncbi.nlm.nih.gov/32054475)

Schwarz B, **Azodi CB**, **Shiu SH**, Bauer P  (2020) Putative cis-regulatory elements predict iron deficiency responses in *Arabidopsis* roots. *Plant Physiol* 182(3):1420 [pubmed](https://pubmed.ncbi.nlm.nih.gov/31937681)

**Azodi CB**, Pardo J, VanBuren R, de los Campos G, **Shiu SH** (2020) Transcriptome-based prediction of complex traits in maize. *Plant Cell* 32:139-151 [pubmed](https://www.ncbi.nlm.nih.gov/pubmed/31641024)

## 2019

**Azodi CB, Bolger E<font color="red"><sup>u</sup></font>**, McCarren A, Roantree M, de los Campos G, **Shiu SH** (2019) Benchmarking algorithms for genomic prediction of complex traits. *G3* 9(11):3691 [pubmed](https://www.ncbi.nlm.nih.gov/pubmed/31533955)

**Uygun S<font color="green"><sup>e</sup></font>**, **Azodi, CB<font color="green"><sup>e</sup></font>**, **Shiu, SH** (2019) Cis-regulatory code for predicting plant cell-type specific high salinity response. *Plant Physiol.* 181(4):1739 [pubmed](https://www.ncbi.nlm.nih.gov/pubmed/31551359)

**Lloyd JP**, Bowman MJ, **Azodi CB**, Moghe GD, Childs KL, **Shiu SH**. Evolutionary characteristics of intergenic transcribed regions indicate rare novel genes and widespread noisy transcription in the Poaceae (2019) *Scientific Reports* 9:12122 [pubmed](https://www.ncbi.nlm.nih.gov/pubmed/31431676)

**Panchy NL, Azodi CB, Winship EF<font color="red"><sup>u</sup></font>, Shiu SH** (2019) Expression and regulatory asymmetry of retained *Arabidopsis thaliana* transcription factor genes derived from whole genome duplication. *BMC Genomics* 19:77 [pubmed](https://www.ncbi.nlm.nih.gov/pubmed/30866803)

Chang YM, Lin HH, Liu WY, Yu CP, Chen HJ, Kao YY, Wu YH, Lin JJ, Li MYJ, Tu SL, Wu SH, **Shiu SH**, Ku MSB, Li WH (2019) A comparative transcriptomics method to infer gene coexpression networks and its applications to maize and rice leaf transcriptomes. *Proc. Natl. Acad. Sci., USA* 116(8):3091-3099 [pubmed](https://www.ncbi.nlm.nih.gov/pubmed/30718437)

**Moore BM**, **Wang P**, Fan P, Leong B, Schenck C, **Lloyd J**, **Lehti-Shiu MD**, Last R, Pichersky E, **Shiu SH** (2019) Robust predictions of specialized metabolism genes through machine learning. *Proc. Natl. Acad. Sci., USA* 116(6):2344-2353 [pubmed](https://www.ncbi.nlm.nih.gov/pubmed/30674669)


[Click here for publications >5 years old](https://shiulab.github.io/publications_older/)
